### PR TITLE
Add accessibility section

### LINKED ISSUE #324: Add accessibility section
In the [WebApps charter](https://www.w3.org/2019/05/webapps-charter.html#scope) it says:
>Where there are implications for implementors, developers, or users, in the areas of accessibility, internationalization, privacy, and security, each specification must have a section that describes relevant benefits, limitations, and best practice solutions for that particular area.>

To this end each spec needs to have an accessibility section. For the avoidance of doubt in the reader, the section should be included even if it indicates there are no known accessibility considerations.

Relates to w3c/webappswg#15

### PR DESCRIPTION
**Pull request by marcoscaceres**
Closes #324

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/push-api/pull/342.html" title="Last updated on Oct 25, 2021, 7:35 AM UTC (75398ee)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/push-api/342/95c742d...75398ee.html" title="Last updated on Oct 25, 2021, 7:35 AM UTC (75398ee)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1273,6 +1273,19 @@ <h2>
...
+    <section>
+      <h2>
+        Accessibility
+      </h2>
+      <p>
+        The <cite>Push API</cite> does not provide any means itself to present data it receives
+        from a [=push service=]. Instead, the <cite>Push API</cite> relies on other APIs –
+        primarily the [[[NOTIFICATIONS]]] – to present received information to an end user. As
+        such, there are no accessibility requirements for the <cite>Push API</cite> itself.
+        However, specifications such as the [[NOTIFICATIONS]] provide guidance how to present
```
Comment (martinthomson) [inline index.html]:
```suggestion
        However, specifications such as [[NOTIFICATIONS]] provide guidance how to present
```

**Review by martinthomson — APPROVED** [review]
Any sense in acknowledging that the amount of information the API conveys is limited, so sites cannot send any larger resources that a11y might depend on over the push message channel; those need to be fetched separately.

**Comment by marcoscaceres** [comment]
yeah, saying that could make sense.

**Comment by martinthomson** [comment]
How about:

> Providing an interface with good accessibility might depend on transferring more information than a push message can convey. Push messages are best suited to carrying small amounts of content or identifiers.  Any larger resources need to be fetched from servers.

**Pull request merged**